catalog/opensearch/catalog-opensearch-source/src/main/java/org/codice/ddf/opensearch/source/OpenSearchSource.java

```diff
@@ -168,16 +175,23 @@
 
   protected final OpenSearchFilterVisitor openSearchFilterVisitor;
 
-  protected Integer connectionTimeout;
+  protected Integer connectionTimeout = 30000;
```

Comment:
❓ Should these be configurable? Any reason you chose 30000 and 60000?

Reply:
those are the default values. I set them b/c blueprint sets the properties onesie twosies and I create the `factory` object with around five fields. all of the weird things I don't like about this PR are things that I couldn't figure out a better way to do w/ blueprint